src/components/swipe/style/swipe.less

```diff
@@ -6,7 +6,7 @@
     height: 100%;
     top: 0px;
     left: 50%;
-    z-index: 1000; 
+    z-index: 990; 

```

Comment:
I think it's better to have the swip under copyrigth, button, scale bar and links. They all have ``z-index: 1000`